src/Parameters/JoinMeetingParameters.php

```diff
@@ -225,4 +223,34 @@ public function setUserId(string $userID)
 
         return $this;
     }
+
+    /**
+     * @deprecated since version 4.3 and will be removed in version 5.0. The API parameter was removed from BigBlueButton and has no effect anymore.
+     *
+     * @return $this
+     */
+    public function setJoinViaHtml5(bool $joinViaHtml5): self
+    {
+        @trigger_error(
+            sprintf('Using "%s" is deprecated since version 4.3 and will be removed in version 5.0. The API parameter was removed from BigBlueButton and has no effect anymore.', __METHOD__),
```

Comment:
```suggestion
            sprintf('Using "%s()" is deprecated since version 4.3 and will be removed in version 5.0. The API parameter was removed from BigBlueButton and has no effect anymore.', __METHOD__),
```